Bottom: c326e353647f879be6ccf2f210b537d6aeee3580
Top:    52070af12d011fdc659b828cb5c7689796a3e1a9
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-12 17:14:04 +0800

Refresh of autofs-5.0.7-fix-probe-each-nfs-version-in-turn-for-singleton-mounts.patch

---

diff --git a/modules/mount_nfs.c b/modules/mount_nfs.c
index 81ba3ca..9de8a73 100644
--- a/modules/mount_nfs.c
+++ b/modules/mount_nfs.c
@@ -190,7 +190,7 @@ int mount_mount(struct autofs_point *ap, const char *root, const char *name, int
 	 * to NFSv3 (if it can). If the NFSv4 probe fails then probe as
 	 * normal.
 	 */
-	if (!hosts->next &&
+	if ((hosts && !hosts->next) &&
 	    mount_default_proto == 4 &&
 	    vers & NFS_VERS_MASK != 0 &&
 	    vers & NFS4_VERS_MASK != 0) {
